simulation-runner/config/nodes/ops-node.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
This file is likely incomplete and will need to be updated once we're able to get the scala simulation further in its execution.

Particularly, I'm not sure if the `simulation_class` will work or if we need to make a new PE recording, or maybe use the `WithFeeder` version.  We may want to use a different catalog zero module too; I'm not sure that I got far enough to know if this is the correct one.